app/_how-tos/use-ai-prompt-guard-plugin.md

```diff
@@ -0,0 +1,213 @@
+  a: Use the AI Prompt Guard plugin with regex patterns to allow or deny prompts based on user prompts.
+
+tools:
+  - deck
+
+prereqs:
+  inline:
+    - title: Mistral
+      include_content: prereqs/mistral
+      icon_url: /assets/icons/mistral.svg
+  entities:
+    services:
+      - example-service
+    routes:
+      - example-route
+
+cleanup:
+  inline:
+    - title: Clean up Konnect environment
+      include_content: cleanup/platform/konnect
+      icon_url: /assets/icons/gateway.svg
+    - title: Destroy the {{site.base_gateway}} container
+      include_content: cleanup/products/gateway
+      icon_url: /assets/icons/gateway.svg
+---
+
+## Configure the AI Proxy plugin
+
+Start by configuring the AI Proxy plugin to route prompts to Mistral AI.
+
+{% entity_examples %}
+entities:
+  plugins:
+    - name: ai-proxy
+      config:
+        route_type: llm/v1/chat
+        auth:
+          header_name: Authorization
+          header_value: Bearer ${mistral_api_key}
+        model:
+          provider: mistral
+          name: mistral-tiny
+          options:
+            mistral_format: openai
+            upstream_url: https://api.mistral.ai/v1/chat/completions
+variables:
+  mistral_api_key:
+    value: $MISTRAL_API_KEY
+{% endentity_examples %}
+
+## Configure the AI Prompt Guard plugin
+
+Next, configure the AI Prompt Guard plugin to allow general IT and helpdesk questions while denying prompts related to hacking, phishing, or inappropriate content.
+
+{% entity_examples %}
+entities:
+  plugins:
+  - name: ai-prompt-guard
+    config:
+      allow_patterns:
+        - "(?i).*what is .*"
+        - "(?i).*how do i .*"
+        - "(?i).*install .*"
+        - "(?i).*configure .*"
+        - "(?i).*reset .*"
+        - "(?i).*troubleshoot .*"
+      deny_patterns:
+        - "(?i).*bypass.*(login|password|auth).*"
+        - "(?i).*hack.*"
+        - "(?i).*phish.*"
+        - "(?i).*malware.*"
+        - "(?i).*cve.*"
+        - "(?i).*exploit.*"
+        - "(?i).*social engineering.*"
+        - "(?i).*pentest.*"
+        - "(?i).*impersonate.*"
+        - "(?i).*dating.*"
+{% endentity_examples %}
+
+## Validate configuration
+
+Use sample prompts to confirm that allowed categories (general IT questions) pass through while unsafe or irrelevant requests are blocked.
+
+{% navtabs "pattern-prompt-guard-it-tests" %}
+{% navtab "Allowed: General IT questions" %}
+
+These prompts match `allow_patterns` and should succeed:
+
+{% validation request-check %}
+url: /anything
+headers:
+  - 'Content-Type: application/json'
+  - 'Authorization: Bearer $DECK_OPENAI_API_KEY'
+body:
+  messages:
+    - role: user
+      content: What is DNS?
+{% endvalidation %}
+
+{% validation request-check %}
+url: /anything
+headers:
+  - 'Content-Type: application/json'
+  - 'Authorization: Bearer $DECK_OPENAI_API_KEY'
+body:
+  messages:
+    - role: user
+      content: How do I install Python on Windows?
+{% endvalidation %}
```

Comment:
```suggestion
```
I'm wondering if we should just have the one example per tab, otherwise it looks a bit off in formatting to have one request right after another.